packages/lib/server/repository/user.ts

```diff
@@ -435,4 +441,52 @@ export class UserRepository {
       },
     });
   }
+
+  static async create({
```

Comment:
**Blocking**: I feel like we should abstract this and the logic that's found in the user signup. At least abstract the very basic info that's needed to be written in the DB. I'm not a fan of having two different areas for user creation.

I'm open to discussion here.

Reply:
I have abstracted this here in UserRepository which I believe is the right place.

What remains is refactoring other places that's doing `prisma.user.create` to reuse this method which I think would be out of scope for this PR.

Let me know if you disagree.

Reply:
Thinking about it, your right I wouldn't want to add that to this PR. Added the issue here: https://github.com/calcom/cal.com/issues/14869